### DISCUSSION TITLE
Should we let the API be public or restrict them (at least for the first release) to the Frontend / aggregator that will consume them?

### DISCUSSION THREAD

**Discussion by grugna** [CBIIT/ccdi-federation-api#22] [API Design Discussion]
I personally think at least for the first version we should restrict access to the API to the aggregator / frontend application Lidos is working on. We haven't discussed much about potential malicious usage of the API by unknown third party just trying to get all the data outside the system. 
I am not sure if the frontend application will require any login, so this may just mean move the problem one level up and not resolving it.

**Comment by claymcleod** [comment]
I have a slight preference to make the API open from the start, but I don't really have any justification for that beyond the fact that it would be nice to get our work out into the public and make an impact ASAP.
